front-end/src/styles/table.scss

```diff
@@ -1,8 +1,9 @@
 @import '_variables.scss';
 
 .table {
-  height: 85vh;
+  // height: 85vh;
```

Comment:
this obsolete?

Reply:
lol yes. good catch ;) was experimenting and liked the no-fixed-height better